### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by tirphana** [downthemall/downthemall#366]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to manually navigate to the DMA toolbar icon and choose everything click by click when dealing with multiple tabs/windows that need downloading.

**Describe the solution you'd like**
What I would want to happen is that keyboard shortcuts for the basic functions such as "DownThemAll!" and "DownThemAll! - All tabs" are implemented inside the browser and are accessible for customization (through the shortcuts section of about:addons, in Firefox at least).

**Describe alternatives you've considered**
I don't really see any alternative to this, aside from maybe trying other similar addons, but I haven't bothered so far.

**Additional context**
I don't think any is necessary, but I can provide more if needed.

**Comment by jwilk4** [comment]
I'd vote for this, and suggest including a keyboard shortcut for DTA-One-Click. 

There also used to be an option to add a second toolbar button for DTA and you oould pick which DTA function it would call (mgr, one-click, etc).   I don't know if we could get that back, but it would supplement this in adding UI efficiency and ease of use.

**Comment by orangpelupa** [comment]
yeah a down them all one-click keyboard hotkey (or custom binding)  would be amazing indeed.

**Comment by deathbird909** [comment]
I agree that this would make an already useful extension even more useful.
